test/sqllogictest/redacted.slt

```diff
@@ -223,8 +223,5 @@ SHOW REDACTED CREATE MATERIALIZED VIEW v;
 query error db error: ERROR: materialize\.public\.mv1 is not a view
 SHOW REDACTED CREATE VIEW mv1;
 
-query error Expected end of statement, found TYPE
-SHOW CREATE TYPE ty;
-
 query error Expected end of statement, found TYPE
 SHOW REDACTED CREATE TYPE ty;
```

Comment:
In CI, "Fast SQL logic tests 4" is failing because this parsed successfully, but the test expects an error. This is because `parse_show` starts with parsing the `REDACTED` keyword.

Reply:
Can I get more elaboration on that? 

I see this `parse_show` fn inside `src/sql-parser/src/parser.rs`
```rust
fn parse_show(&mut self) -> Result<ShowStatement<Raw>, ParserError> {
        let redacted = self.parse_keyword(REDACTED);
        ...
```

This shouldn't throw an error if REDACTED isn't present so `SHOW CREATE TYPE` should work. What else can be the reason the test expects an error? Maybe I missed something in the slt `sqllogictest` format ?

Reply:
Done